Dockerfile

```
@@ -13,7 +13,7 @@ RUN install-php-extensions \
 
 COPY --from=wordpress /usr/local/etc/php/conf.d/* /usr/local/etc/php/conf.d/
 COPY --from=wordpress /usr/local/bin/docker-entrypoint.sh /usr/local/bin/
-COPY --from=wordpress --chown=www-data:www-data /usr/src/wordpress /app/public
+COPY --from=wordpress --chown=root:root /usr/src/wordpress /app/public
```

```suggestion
COPY --from=wordpress /usr/src/wordpress /app/public
```

Should be enough, isn't it?

---

no, because the files from the wordpress image are also owned by www-data:www-data and it didn't work for me in my test.